test/metric_value_benchmark/eks_resources/test_schemas/container_efa.json

```diff
@@ -6,9 +6,9 @@
   "properties": {
     "CloudWatchMetrics": {},
     "ClusterName": {},
-    "EfaDevice": {},
```

Comment:
Did we previously have EfaDevice as a dimension if the tests had mentions of it here?